docs/internals/stream_checkpointing.zh.md

```diff
@@ -23,151 +23,117 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This document describes Flink's fault tolerance mechanism for streaming data flows.
+本文将介绍 Flink 流数据处理的容错机制（fault tolerance mechanism）。
```

Comment:
seems that this page is not in this issue， maybe you need to open another pr for this.